File: 01-Introduction-and-Installation/cv2_read_image_time.cpp
#include "stdafx.h"
#include <opencv2\opencv.hpp>

using namespace cv;
using namespace std;

int main()
{
	// 开始计时
	int start = getTickCount();

	// 读入一张图片
	Mat gSrcImage = imread("lena.jpg");

	// 停止计时
	int end = getTickCount();

	// 单位：s
	double time = (end - start) / getTickFrequency();
	printf("%f", time);

	getchar();
    return 0;	
}

File: 01-Introduction-and-Installation/cv2_read_image_time.py
import cv2

# 开始计时
start = cv2.getTickCount()

# 读入一张图片
# You should use Flag here so that it will read colored or grey 
img = cv2.imread('lena.jpg', 0)

# 停止计时
end = cv2.getTickCount()

# 单位：s
print((end - start) / cv2.getTickFrequency())
